setup.cfg

```diff
@@ -2,6 +2,7 @@
 indent='    '
 line_length = 120
 multi_line_output=3
+known_first_party=code_annotations
```

Comment:
I'm not sure this is the right thing to do here, but it was the only way I could find to get `isort` to recognize that imports from `code_annotations` were in the correct place.

Reply:
https://github.com/search?q=org%3Aedx+known_first_party&type=Code

Looks like it's done in several other repos.